### DISCUSSION TITLE
Caso Pastelería ¿Se realizará caso de estudio?

### DISCUSSION THREAD

**Discussion by Gracesl** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#179] [Q&A]
¿Alguien hizo caso de estudio del caso de la pastelría con las herramientas que nos proporcionaron en la Intro de Frontend?

**Comment by doguedogue** [comment]
Puedes hacerlo aunque no es obligatorio dijo el MC

**Comment by Ingrzs** [comment]
No, solo es puro codigo, el caso de estudio no es obligatorio pero en la vida profesional laboral siempre se hace el análisis, el primero se hizo para que nosotros mismos siempre viéramos como son los procesos de creación de un sistema antes de empezar con el código.

**Comment by Gracesl** [comment]
He estado leyendo blogs de maquetación de página web, yo creo plasmare en un documento junto con wireframes. Gracias compañeros.

**Comment by JuanSalvador73** [comment]
No es necesario el caso de estudio, lo que si recomiendo es el diagrama UX del sitio, aunque las especificaciones dadas lo presentan como un proyecto "simple", literalmente solo a entregar un HTML sencillo, al menos en esta etapa. 

Pero el MC nos dió libertad de incluir lo que queramos, incluso tópicos de FrontEnd que aún no hemos visto, si tenemos conocimientos mas adelantados. Es a tu gusto.

Saludos.

**Comment by ChristianDeM** [comment]
Practiamente se esta trabajando un portafolio con nuestras cuentas de git. cualquier info extra es un plus para tu cuenta de git.
